users/class_tests/login_tests.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from users.models import User
+from collections import namedtuple
+from django.contrib.auth import authenticate
+from pytest_django.asserts import assertTemplateUsed
+from users.forms import MyUserCreationForm as LoginForm
+
+
+USER_CREDENTIALS = namedtuple("USER_CREDENTIALS", "email password")
+
+
+@pytest.mark.django_db
+class TestLogin:
+
+    @pytest.fixture
+    def signed_up_credentials(self):
+        User.objects.create_user(username='valid_username', email='valid@mta.ac.il', password='pw123123')
```

Comment:
Please use constants like:
```suggestion
VALID_USER_NAME = 'valid_username'
USER_EMAIL  = 'valid@mta.ac.il'
USER_PASS = 'pw123123'
        User.objects.create_user(username=VALID_USER_NAME, email=USER_EMAIL, password=USER_PASS)
```
like this it will be:
1. easy to maintain the code
2. prevent typo errors/mistakes

Reply:
done. thanks 😄